PolyEngine/Engine/Src/InputSystem.cpp

```diff
@@ -14,6 +14,12 @@ void InputSystem::InputPhase(World* world)
 	com->PrevMouse = com->CurrMouse;
 	com->PrevWheel = com->CurrWheel;
 
+    for (auto it=com->Controllers.begin(); it!=com->Controllers.end(); ++it)
```

Comment:
intead of begine end loop, use for each loop like so: `for (Controller& controller : Controllers)`